nav2_system_tests/urdf/turtlebot3_waffle.urdf

```diff
@@ -220,9 +261,9 @@
   </link>
 
   <joint name="camera_rgb_joint" type="fixed">
-    <origin xyz="0.005 0.018 0.013" rpy="0 0 0"/>
+    <origin xyz="0.005 0.018 0.013" rpy="-1.57 0 -1.57"/>
     <!-- <origin xyz="${r200_cam_rgb_px} ${r200_cam_rgb_py} ${r200_cam_rgb_pz}" rpy="0 0 0"/> -->
-    <parent link="camera_link"/>
+    <parent link="realsense_link"/>
```

Comment:
Aight, I'm OK with adding references to realsense where there wasn't something before, but don't change things to release to un-generalize things that existed. Please change change to camera link

Reply:
Actually, the link names in the Gazebo model and urdf need to match. That is why I changed it to realsense link. Since I changed the parent link name to realsense_link, I decided to use the same reference in all the child frame names for depth camera.

Reply:
So change the sdf file?

Reply:
ok got it. remove -all realsense :)

Reply:
I guess the robot color can stay blue :D